recipes/zeromq/zeromq.inc

```diff
@@ -0,0 +1,21 @@
+inherit autotools pkgconfig c++ auto-package-utils auto-package-libs
+
+RECIPE_TYPES = "machine sdk"
```

Comment:
Huh, if you build an application using "the sdk", that application is for target/MACHINE, so you would need to link it with machine:libzmq.